helm/prow-log-aggregator/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: prow
+  name: log-aggregator
+  labels:
+    app: log-aggregator
+spec:
+  replicas: 1
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: log-aggregator
+  template:
+    metadata:
+      labels:
+        app: log-aggregator
+    spec:
+      serviceAccountName: log-aggregator
+      containers:
+        - name: log-aggregator
+          image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+          args:
+            - -namespace
```

Comment:
Double checking the single dash here - should it be `--namespace`?

Reply:
I am also not sure - this is a straight copy of @tfussell code :D

Reply:
This is using the standard library flags package which has only one dash before long flags rather than `viper` which uses two. Just a quick hack--if it's going to be its own thing we should use the whole viper/cobra setup.